dashboard/app/models/script.rb

```diff
@@ -1826,9 +1826,8 @@ def get_feedback_for_section(section)
 
     feedback = {}
 
-    level_ids = script_levels.map(&:oldest_active_level).select(&:can_have_feedback?).map(&:id)
     student_ids = section.students.map(&:id)
-    all_feedback = TeacherFeedback.get_all_feedback_for_section(student_ids, level_ids, section.user_id)
+    all_feedback = TeacherFeedback.get_student_feedbacks_for_script(id, student_ids, section.user_id)
```

Comment:
(unrelated to this change) Any idea why this method (`get_feedback_for_section`) is in the `script` model?

Reply:
No idea!